src/ActionList/Item.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import styled from 'styled-components'
+import {AlertIcon} from '@primer/octicons-react'
 import Box, {BoxProps} from '../Box'
+import {Tooltip} from '../drafts'
```

Comment:
I'm using the new, accessible Tooltip. Is this safe to use even though it's still in the experimental phase?

Reply:
cc @broccolinisoup

Reply:
In terms of using the new draft Tooltip, I think it is safe. It is unlikely to have API changes but still possible since it is in the experimental stage but it wouldn't be a big deal for us anyway because we could always update the internals if Tooltip API is changed. 

Tooltip is still going through the accessibility sign-off review but compared to the stable one it solves multiple problems and it is more accessible. I think trying out the new tooltip in Primer rather than dotcom first seems like a really good idea to me.

Reply:
Posted a comment/question about the usage as well https://github.com/primer/react/pull/3913#discussion_r1386025044 - very keen to hear what you think!